packages/flutter/test/painting/text_painter_test.dart

```diff
@@ -1259,22 +1259,22 @@ void main() {
       WidgetSpan(child: Text('widget')),
       TextSpan(text: 'after'),
     ]);
-    expect(painter.plainText, 'before\n\uFFFCafter');
+    expect(painter.plainText, 'before\n${String.fromCharCode(Unicode.objReplacementChar)}after');
```

Comment:
Definitely good to be replacing the magic number here and the places like it.

Should it be a direct reference to [Unicode.objReplacementChar], or should it be to [PlaceholderSpan.placeholderCodeUnit]? I haven't studied the code around this, but it kind of looks like it's in an area where the reason it's U+FFFC would be because that's what [PlaceholderSpan] and friends use.